catroid/src/main/java/org/catrobat/catroid/stage/StageActivity.java

```diff
@@ -233,6 +233,7 @@ public void onBackPressed() {
 			finish();
 		} else {
 			StageLifeCycleController.stagePause(this);
+			stageListener.makeScreenshot(false);
```

Comment:
take an automatic screenshot after pausing the stage before showing the stage dialog. I guess this is the best place for it.